rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,208 @@
+
+When a container is created, associated volumes enter `activating` state. Once the container enters the `running` phase, its volumes enter `active` state. Containers attaching to an already `active` volume will not impact that volume's state.
+
+When all containers attached to a volume are marked for deletion, the volume enters a `dactivating` state. Once the conatiners are deleted, the volume enters `detached` state.
+
+When a volume is marked for deletion, it enters a `removing` state. Once the data is deleted from the host(s), it enters `removed` state. Removed volumes will not appear when listing volumes, but they continue to exist in Rancher API for a period of time for debugging and auditing purposes.
+
+### Volume Scopes
+
+There are two different volume scopes. The scope refers to the level at which the volume is managed by Rancher.
+
+#### Environment Scope
+
+An `environment scope` volume may be shared by all containers in an environment. Rancher schedules containers appropriately based on the hosts that have access to its assigned volumes.
+
+Rancher requires environment volumes to already exist prior to creating a service consuming it. Any installed volume driver can be used.
+
+The practical benefit of this scope is the ability to easily share data between different software services/stacks whose lifecycles are independently managed. The user is given full control and must [manage the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#managing-volumes) themselves.
+
+##### Shared Environment Scope Volume Example
+
+First, [create the environment scope volume]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#create-a-volume) to be shared between stacks.
+
+```
+$ rancher volume create --driver local redis-data-external
+```
+
+Define the volume with `external: true` modifier to indicate the volume's lifecycle is managed externally (environment scope). Save the `docker-compose.yml` file below to a directory.
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: redis:3.0.7
+    volumes:
+    - redis-data-external:/data
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+Now create the stack.
+
+```
+$ rancher stack create
+```
+
+[List the volumes]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/volumes/#list-volumes) to verify that `redis-data-external` volume is `active`.
+
+> **Note:** Scaling a service up or down mounts or unmounts the same shared volume.
+
+Any new stack within the environment may attach to the same `redis-data-external` volume. The easiest way is to copy the volume definition verbatim.
+
+```yaml
+volumes:
+  redis-data-external:
+    driver: local
+    external: true
+```
+
+#### Stack Scope
+
+A stack-scoped volume is owned and managed by the stack in which it is defined. The primary benefit is the volume lifecycle is automatically managed by Rancher as part of the stack lifecycle.
```

Comment:
I stumbled reading this. I think "The primary benefit is **that** the volume lifecycle" would clear it up.